bokehjs/test/unit/models/tools/inspectors/hover_tool.ts

```diff
@@ -74,6 +76,32 @@ describe("HoverTool", () => {
     })
   })
 
+  describe("line_policy", () => {
+    it("should support 'none' line_policy with mode=vline", async () => {
+      const tooltips: [string, string][] = [["x", "$x"], ["y", "$y"]]
+      const hover = new HoverTool({mode: "vline", line_policy: "none", tooltips})
+      const p = fig([200, 200], {toolbar_location: null, min_border: 0, x_range: [1, 3], y_range: [0, 2]})
+      const r = p.line([1, 2, 3], [1, 1, 1])
+      p.add_tools(hover)
+
+      const {view} = await display(p)
+
+      const crv = view.renderer_views.get(r)!
+      const [[sx], [sy]] = crv.coordinates.map_to_screen([1.8], [1.5])
+      const ui = view.canvas_view.ui_event_bus
+      const {left, top} = offset_bbox(ui.hit_area)
+      const ev = new MouseEvent("mousemove", {clientX: left + sx, clientY: top + sy})
+      ui.hit_area.dispatchEvent(ev)
+
+      await view.ready
+
+      const hover_view = view.tool_views.get(hover)! as HoverTool["__view_type__"]
```

Comment:
Use `ViewManager` instead like this `view.owner.get_one(hover)`.

Reply:
https://github.com/bokeh/bokeh/pull/12586/commits/757f67a325cf11702c68e2e4e273252409fad968